developer-docs-site/docs/nodes/validator-node/operator/node-requirements.md

```diff
@@ -55,7 +55,9 @@ On the one hand, network storage requires additional CPU support to scale IOPS,
 
 When you are running a validator node, you are required to open network ports on your node to allow other nodes to connect to you. For fullnodes this is optional.
 
-There are three types of Aptos networks. Your node can be configured so that each of these networks can connect to your node using a different port on your node.
+### Network types
+
+There are three types of Aptos networks. Your node can be configured so that each of these networks can connect to your node using a different port on your node:
```

Comment:
> Make sure to keep the TCP port 6186 open for the local backup storage service.

Can we change this line below as well? -- it sounds like we suggest to open that port.

Reply:
Thanks, @msmouse . Should I remove it entirely? I see no other reference to it on the site.

Reply:
And should I remove it for both node types, @msmouse ?

Reply:
Yes let's remove. When I and @areshand  have a better story around opening the backup & restore tooling to the community, we can add it back properly.

Reply:
Thanks, @msmouse ! All done.